tests/test_buffer.py

```diff
@@ -436,6 +437,11 @@ def test_SimpleArray_from_ndarray(self):
         sarr_from_cpp = modmesh.SimpleArrayFloat64(shape=(2, 3, 4))
         self.assertFalse(sarr_from_cpp.is_from_python)
 
+        shape = (2, 2, 2, 2)
```

Comment:
Could you please also test with the shape in which each element different from others?  `(2, 2, 2, 2)` is too simple.

Reply:
`(2, 3, 5, 7)` looks sufficiently interesting.